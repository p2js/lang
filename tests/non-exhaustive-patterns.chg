type Unnested = enum { A, B, C };

fn main(): i32 {
    let x = Unnested::A;
    match x {
        Unnested::A => { return 1; }
        Unnested::B => { return 2; }
    };
}
