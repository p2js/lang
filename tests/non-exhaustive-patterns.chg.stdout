expr @ 4:23 4:24 => Unnested
expr @ 5:11 5:12 => Unnested
expr @ 6:33 6:34 => i32
expr @ 6:26 6:34 => never
expr @ 6:24 7:0 => ()
expr @ 7:33 7:34 => i32
expr @ 7:26 7:34 => never
expr @ 7:24 8:0 => ()
expr @ 5:5 8:6 => ()
expr @ 3:16 10:1 => ()

